modules/release-notes/con-relnotes-known-issues.adoc

```diff
@@ -22,3 +22,12 @@ If you deploy {product-short} without a proxy, there is the potential for IP add
 * Prevent collection of IP addresses by configuring a reverse proxy in front of {product-short}.
 //link:{LinkRHIDPIssue}RHIDP-2766[RHIDP-2766]
 --
+
+Rollbacks are not supported::
++
+--
+If you deploy {product-short} using either Helm or the Operator, bear in mind that rollbacks are not supported.
```

Comment:
```suggestion
If you deploy {product-short} using either Helm or the Operator, rollbacks to earlier versions are not supported.
```
We should specify the platform that we are deploying RHDH on. Is this specific to OCP or also true for other K8s engines?

Reply:
This affects all platforms, not only OCP.